lochness/keyring/__init__.py

```diff
@@ -40,11 +40,16 @@ def box_api_token(Lochness, key):
     '''get box api token from keyring'''
     if key not in Lochness['keyring']:
         raise KeyringError('\'{0}\' not in keyring'.format(key))
-    if 'USER_ID' not in Lochness['keyring'][key]:
-        raise KeyringError('\'USER_ID\' not in {0}'.format(key))
+    if 'ENTERPRISE_ID' not in Lochness['keyring'][key]:
+        Lochness['keyring'][key]['ENTERPRISE_ID'] = 'NOTGIVEN'
```

Comment:
I see that you removed `raise`. So can lochness now proceed with `NOTGIVEN` `USER_ID`?

Reply:
Do you remember the issue we had with `Box app authentication`? During resolving this issue, I had tried switching back and forth between different `box API` options, in which I also tried both `USER_ID` and `ENTERPRISE_ID`. Now the current version is set to use `ENTERPRISE_ID`, and I'll add back `raise` functionality soon.

Reply:
4dbb437